packages/extension-browser/tests/devtools.ts

```diff
@@ -98,7 +98,8 @@ test('It builds a configuration, starts a scan, and displays results', async (t)
      * Launch the browser and get a reference to the initial page.
      * Note: Uncomment config options to see page content (aids debugging).
      */
-    const browser = await launch(/* { defaultViewport: null, headless: false } */);
+    const executablePath = chromiumFinder.getInstallationPath();
+    const browser = await launch(/* { defaultViewport: null, headless: false } */{ executablePath});
```

Comment:
```suggestion
    const browser = await launch(/* { defaultViewport: null, headless: false } */{ executablePath });
```

Reply:
Need to find a setting to make sure this doesn't happen. We've had quite a few of these in the past and eslint doesn't flag them 🤔